### DISCUSSION TITLE
为什么我的清爽首页夜间模式分区排行榜字体颜色一直是黑色的？

### DISCUSSION THREAD

**Discussion by Alexius66** [the1812/Bilibili-Evolved#3769] [提问 / Q&A] [answered]
![image](https://user-images.githubusercontent.com/73510996/197344389-4a67ddb8-07ac-4c96-9bab-117d14c9a267.png)

**Comment by the1812** [comment, marked as answer]
你组件没更新吧
![image](https://user-images.githubusercontent.com/26504152/197813734-1dc487ea-f6d9-4db3-9cb9-12c37c1785e6.png)

**Reply by Alexius66** [reply]
> 你组件没更新吧 ![image](https://user-images.githubusercontent.com/26504152/197813734-1dc487ea-f6d9-4db3-9cb9-12c37c1785e6.png)

问题解决了，好像是浏览器的安全设置改到了高，使得脚本没有权限改某些东西导致的，重置浏览器以后就解决了。奇奇怪怪的
